airflow/contrib/hooks/gcp_dataflow_hook.py

```diff
@@ -404,6 +405,7 @@ def start_python_dataflow(
         variables: Dict,
         dataflow: str,
         py_options: List[str],
+        py_interpreter=None,
```

Comment:
```suggestion
        py_interpreter: str = 'python2',
```
All gcp _ * _ hook.py hooks use typehint to make them easier to use. The default value passed here will make the code easier to understand.